skills/lark-workflow-okr-weekly/SKILL.md

```
@@ -0,0 +1,210 @@
+
+```bash
+lark-cli auth login --domain okr,task            # 基础（读 OKR + 任务）
+lark-cli auth login --domain okr,task,docs,drive # 含写出周报为飞书文档（可选）
+lark-cli auth login --domain okr,task,im         # 含发送到群（可选）
+```
+
+**租户必须启用 OKR 产品**，否则 `okr +cycle-list` 会返回空或无权限错误。
+
+## 工作流
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)